pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1001,6 +1001,7 @@ type httpListenerOpts struct {
 
 	suppressEnvoyDebugHeaders bool
 	skipIstioMXHeaders        bool
+	isGatewayMtls             bool
```

Comment:
add comments. it is istio_mutual

Reply:
Added a comment and renamed the var to explicitly mentioned Istio mTLS.